EnglishFilter/sections/css_extended.txt

```diff
@@ -1714,6 +1714,7 @@ urbanthesaurus.org#?#.words > div[style]:not([class]):has(> .ad-tag)
 mynewsmedia.co#?#center:has(> iframe.exclude-popad)
 forum.ragezone.com#?##posts > li.postcontainer:has(> div.postdetails span.usertitle:contains(Check out our sponsor))
 udemyfreecourses.org#?#h4:contains(Sponsored ads)
+freesoft.id#?#.entry-content > div[class]:has(> p[style] > button)
```

Comment:
You can make more accurate `freesoft.id#?#.entry-content > p ~ div[class]:has(> p[style] > button[onclick*="https://href.li/"])`